src/charm.py

```diff
@@ -499,7 +499,9 @@ def _on_config_changed(self, _event: ConfigChangedEvent) -> None:
         self._refresh_firewall()
         try:
             self._event_timer.ensure_event_timer(
-                "reconcile-runners", self.config["reconcile-interval"]
+                event_name="reconcile-runners",
+                interval=int(self.config["reconcile-interval"]),
+                timeout=int(self.config["reconcile-interval"]) - 1,
```

Comment:
Is it really safe to assume that we do not need a pause of more than one second between two reconciliations?

Reply:
The unit is 1 minutes, which I assume is enough.

Reply:
For interval, its minutes, for timeout, its actual seconds. 

```
man timeout

DURATION is a floating point number with an optional suffix: 's' for seconds (the default), 'm' for minutes, 'h' for hours or 'd' for days.  A duration of 0 disables the associated timeout
```

Reply:
I update the code to be more consistent.
The `interval` was in minutes, but the `timeout` was in seconds.
Now both are in minutes.